adcm_aio_client/core/errors.py

```diff
@@ -15,6 +15,24 @@ class ADCMClientError(Exception):
     pass
 
 
+# Session
+
+
+class ClientInitError(ADCMClientError):
+    pass
+
+
+# Version
+
+
+class VersionRetrievalError(ADCMClientError):
```

Comment:
Is it used anywhere?

Reply:
No, it isn't.

Leftover when I was moving it.

I suggest removing it either if I'll change anything else in this PR 
OR when we'll review all errors.